src/oemof/solph/flows/_flow.py

```diff
@@ -405,37 +421,45 @@ def _create(self, group=None):
 
         # ######################### CONSTRAINTS ###############################
 
-        def _flow_summed_max_rule(model):
+        def _flow_max_capacity_factor_rule(model):
             """Rule definition for build action of max. sum flow constraint."""
-            for inp, out in self.SUMMED_MAX_FLOWS:
+            for inp, out in self.MAX_CAPACITY_FACTOR_FLOWS:
                 lhs = sum(
                     m.flow[inp, out, ts] * m.timeincrement[ts]
                     for ts in m.TIMESTEPS
                 )
                 rhs = (
-                    m.flows[inp, out].summed_max
+                    m.flows[inp, out].max_capacity_factor
                     * m.flows[inp, out].nominal_value
                 )
-                self.summed_max.add((inp, out), lhs <= rhs)
+                self.max_capacity_factor.add((inp, out), lhs <= rhs)
 
-        self.summed_max = Constraint(self.SUMMED_MAX_FLOWS, noruleinit=True)
-        self.summed_max_build = BuildAction(rule=_flow_summed_max_rule)
+        self.max_capacity_factor = Constraint(
+            self.MAX_CAPACITY_FACTOR_FLOWS, noruleinit=True
+        )
+        self.max_capacity_factor_build = BuildAction(
+            rule=_flow_max_capacity_factor_rule
+        )
 
-        def _flow_summed_min_rule(model):
+        def _flow_min_capacity_factor_rule(model):
             """Rule definition for build action of min. sum flow constraint."""
-            for inp, out in self.SUMMED_MIN_FLOWS:
+            for inp, out in self.MIN_CAPACITY_FACTOR_FLOWS:
                 lhs = sum(
                     m.flow[inp, out, ts] * m.timeincrement[ts]
                     for ts in m.TIMESTEPS
                 )
                 rhs = (
-                    m.flows[inp, out].summed_min
+                    m.flows[inp, out].min_capacity_factor
                     * m.flows[inp, out].nominal_value
                 )
```

Comment:
These lines make perfectly clear, what we speak about. LHS is energy (time integral over power), RHS is full load "hours" times nominal power.